src/server/controllers/ApiController.js

```diff
@@ -1,23 +1,43 @@
 import User from '../models/user';
+import bcrypt from 'bcrypt';
```

Comment:
In user.js model, you are using mongoose bcrypt, can it also be used for comparing passwords?

Reply:
Yep, changed